src/main/resources/tags/help/ip.tag

```diff
@@ -0,0 +1,9 @@
+type: text
+aliases: address
+
+---
+Geyser is software that you can download and run yourself, so it doesn't have an IP. That being said, there are a few Geyser-related servies that do have an IP:
+
+- There is an _unofficial_ instance of GeyserConnect (this is probably the easiest way to use Geyser) at `geyserconnect.net` - run `!!geyserconnect` in <#613194762249437245> for more info.
```

Comment:
`(this is probably the easiest way to use Geyser)`

I don't like this - [though by having tags for the unofficial GeyserConnect is an endorsement itself] we shouldn't be super-endorsing what is currently an unaffiliated product. Also, the statement fails to account for plugin versions of Geyser, which are supposed to be the easiest way to set up Geyser for one server.

Reply:
```suggestion
- There is an _unofficial_ instance of GeyserConnect at `geyserconnect.net` - run `!!geyserconnect` in <#613194762249437245> for more info.
```

Reply:
I feel like we should explain something about what GeyserConnect is

Reply:
What about
```diff
+ - There is an _unofficial_ instance of GeyserConnect at `geyserconnect.net` (this allows you to join servers without setting Geyser up yourself) - run `!!geyserconnect` in <#613194762249437245> for more info.
```

Reply:
@rtm516 @Camotoy

Reply:
This is terribly nitpicky, but I don't think we need to add any more details on what it is. Either they're asking for GeyserConnect's IP because they know of its existence or they'll run the command to learn more.

Reply:
This isn’t for people asking for the GeyserConnect IP, this tag is for people who don’t even know what Geyser is and say “HAPP WHAT GEYSER IP”

Reply:
So there’s no way they’ll already know what GeyserConnect is, if they ask for the GeyserConnect IP we can just do `!!geyserconnect`

Reply:
Sometimes people have said "what's the IP" referring to GeyserConnect.

Reply:
That's what `!!geyserconnect` is for though, we already have a tag for that, this is for people who generally ask for an IP without knowing that Geyser is standalone software or about all the different services that have an IP

Reply:
I think it's worth a few extra characters so the tag can serve it's intended purpose a bit better

Reply:
I say we don't do it now and just say "If you're looking for a hosted version of the GeyserConnect software, see `!!geyserconnect`. Especially because there's multiple GeyserConnect IPs already; let's just keep all that information on its own tag.

Reply:
Oh, I forgot that I included the IP, I agree that we shouldn't do that now